src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    : '''
+    Upload the final app output files from /home/dnanexus/out and associate to app output spec field
+    '''
+    mark-section "Uploading final output"
+
+    # create valid empty JSON file for job output, fixes https://github.com/eastgenomics/eggd_tso500/issues/19
+    echo "{}" > job_output.json
+
+    export -f _upload_single_file  # required to be accessible to xargs sub shell
+
+    SECONDS=0
+    find /home/dnanexus/out/ -type f | xargs -P $PROCESSES -n1 -I{} bash -c \
```

Shellcheck

[warning] Use -print0/-0 or -exec + to allow for non-alphanumeric filenames.

(SC2038)